src/components/Dashboard/Dashboard.jsx

```diff
@@ -14,24 +14,20 @@ import PlayedCards from '../Cards/PlayedCards/PlayedCards';
 import axios from 'axios';
 
 export class Dashboard extends React.Component {
-
     constructor(props) {
         super(props);
         this.state = {}
     }
-
```

Comment:
Something small but can you put this whitespace back in? I think it's pretty standard to have a line between each function in a React component